docs/source/faq.rst

```diff
@@ -20,7 +20,7 @@ following in your command line:
 How to upgrade my ASReview installation?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Please see the `upgrade section on the installation page <installation.html#upgrade-asreview>`__.
+Please see :ref:`installation:Upgrade ASReview`.
```

Comment:
@Rensvandeschoot @GerbrichFerdinands @terrymyc @qubixes please see this syntax for cross-referencing. Please use this syntax in future PRs on the docs.